ansible_collections/arista/avd/index.md

```diff
@@ -0,0 +1,85 @@
+- [**arista.avd.cvp_configlet_upload**](roles/cvp_configlet_upload/README.md) - Uploads configlets from a local folder to CloudVision Server.
+- [**arista.avd.eos_validate_state**](roles/eos_validate_state/README.md) - Validate operational states of Arista EOS devices.
+- [**arista.avd.eos_snapshot**](roles/eos_snapshot/README.md) - Collect commands on EOS devices and generate reports.
+- [**arista.avd.dhcp_provisioner**](roles/dhcp_provisioner/README.md) - Configure an ISC-DHCP server to provide ZTP services and CloudVision registration.
+- [**arista.avd.cv_deploy**](roles/cv_deploy/README.md) - Deploys EOS device configurations and tags to the CloudVision management platform.
+
+![Arista AVD Overview](docs/_media/avd_roles_dark.svg#only-dark)
+![Arista AVD Overview](docs/_media/avd_roles_light.svg#only-light)
+
+### Collection installation
+
+Ansible galaxy hosts all stable versions of the `arista.avd` collection. Installation from ansible-galaxy is the most convenient approach for consuming `arista.avd` content. Please follow the collection installation [guide](https://avd.arista.com/stable/docs/installation/collection-installation.html).
+
+### Examples
+
+- [Getting started](https://avd.arista.com/stable/docs/getting-started/intro-to-ansible-and-avd.html)
+- [Arista NetDevOps GitHub repository](https://github.com/aristanetworks/netdevops-examples)
+
+### Custom plugins & modules
+
+See the section "Ansible Collection Plugins" on the menu to the left.
+
+## Additional resources
+
+- Ansible [EOS modules](https://docs.ansible.com/ansible/latest/collections/arista/eos/index.html) on Ansible documentation
+- Ansible [CloudVision modules](https://cvp.avd.sh/en/stable/)
+- [CloudVision Portal](https://www.arista.com/en/products/eos/eos-cloudvision)
+- [Arista Design and Deployment Guides](https://www.arista.com/en/solutions/design-guides)
+
+## Ask a question
+
+Support for the `arista.avd` collection is provided by the community directly in this repository. If you have any questions, please leverage the GitHub [discussions board](https://github.com/aristanetworks/avd/discussions).
+
+### Official Arista support
+
+AVD version 4.x releases with full support from Arista TAC. If your organization has the appropriate support agreement, please don't hesitate to contact TAC with any questions or issues.
+
+## Contributing
+
+Contributing pull requests are gladly welcomed for this repository. If you are planning a big change, please start a discussion first to make sure we'll be able to merge it. Please see [contribution guide](https://avd.arista.com/stable/docs/contribution/overview.html) for additional details.
+
+You can also open an [issue](https://github.com/aristanetworks/avd/issues) to report any problems or submit enhancements.
+
+## License
+
+The project is published under [Apache 2.0 License](https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/LICENSE)
```

Comment:
relative link

Reply:
This will lead to a file download vs a rendering of the license. Thoughts?

Reply:
I'm fine with that.